src/mlpack/core/util/prefixedoutstream.hpp

```diff
@@ -19,7 +19,7 @@
 #include <streambuf>
 #include <stdexcept>
 
-#include <mlpack/core/util/sfinae_utility.hpp>
+#include <mlpack/core/arma_extend/arma_extend.hpp> // Includes Armadillo
```

Comment:
Includes `arma` to use `is_arma_type`, but causes the `pragma` message. Including `prereqs.hpp` or `core.hpp` causes the build to fail (I could not fully comprehend the reason, but it looked like a cyclic dependency).